modules/mixcr/1.0/config/default.yaml

```diff
@@ -0,0 +1,37 @@
+lcr-modules:
+    
+    mixcr:
+
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            sample_fastq_1: "__UPDATE__" 
+            sample_fastq_2: "__UPDATE__"
+
+        scratch_subdirectories: []
+
+        options:
+            mixcr_run: 
+                mrna:
+                    --starting-material rna
+                genome:
+                    --starting-material dna
+
+        # TODO: Add the conda environment once MiXCR is available again - using the jar for now.
+        conda_envs:
+            mixcr: "{MODSDIR}/envs/mixcr"
```

Comment:
This file name should end in .yml

Reply:
MiXCR comes as a jar with an executable script for the time being which I've set to install in `envs\mixcr`. I will remove the TODO for clarity and maybe move the path to the mixcr script under a different tag.

Reply:
If its not a conda environment then you should move this to "params" or, probably better yet, "inputs"